workspaces/lightspeed/plugins/lightspeed/config.d.ts

```diff
@@ -38,5 +38,21 @@ export interface Config {
        */
       url: string;
     }>;
+    samplePrompts?: Array</**
```

Comment:
should we call those `prmopt`? or `query`?
as later on, IIRC,  we may support dynamic system prompt as well

Reply:
Since it is an array, I updated it to `prompts`.